packages/core/useEyeDropper/index.ts

```diff
@@ -1,14 +1,28 @@
 import { ref } from 'vue-demi'
 
-export function useEyeDropper() {
+type WindowEyeDropper = Window & typeof globalThis & {
+  EyeDropper: {
+    new(): {
+      open: () => Promise<{ sRGBHex: string }>
+      [Symbol.toStringTag]: 'EyeDropper'
+    }
+  }
+}
+
+/**
+ * Reactive [EyeDropper API](https://developer.mozilla.org/en-US/docs/Web/API/EyeDropper_API)
+ *
+ * @see https://vueuse.org/useEyeDropper
+ * @param initialValue string
+ */
+export function useEyeDropper(initialValue = '') {
```

Comment:
Prefer an object option here for future extensiablity

Reply:
ok~